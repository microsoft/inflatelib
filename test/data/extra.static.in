
# This test has three blocks, however the last one gets ignored since the second block has 'bfinal' set to true
>1
0       # bfinal = false
01      # Compressed with static codes

# "This is from the first block\n"
>>1
10000100 10011000 10011001 10100011 01010000
10011001 10100011 01010000
10010110 10100010 10011111 10011101 01010000
10100100 10011000 10010101 01010000
10010110 10011001 10100010 10100011 10100100 01010000
10010010 10011100 10011111 10010011 10011011 00111010
0000000

>1
1       # bfinal = true
01      # Compressed with static codes

# "This is from the second block\n"
>>1
0001100 >1 0 >>1 01001 >1 100 >>1
10100011 10010101 10010011 10011111 10011110 10010100
0000101 01001 >1 101 >>1
0000000

# Final, ignored block
>1
1       # bfinal = true
01      # Compressed with static codes

# "This is from the third block\n"
>>1
0001100 >1 0 >>1 01001 >1 101 >>1
10100100 10011000 10011001 10100010 10010100
0000101 01001 >1 100 >>1
0000000
