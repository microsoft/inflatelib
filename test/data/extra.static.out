
"This is from the first block\n"
"This is from the second block\n"
